lib/cartopy/mpl/geoaxes.py

```diff
@@ -417,6 +417,34 @@ def coastlines(self, resolution='110m', color='black', **kwargs):
                                                       resolution, **kwargs)
         return self.add_feature(feature)
 
+    def tissot(self, rad_km=5e5, n_samples=80, lon_n=10, lat_n=6, **kwargs):
+        """
+        Adds Tissot's indicatrices to the current axes with the geodesic class.
+
+        Kwargs:
+
+            * rad_km - The radius in km of the the circles to be drawn.
+
+            * n_samples - Integer number of points sampled around the
+              circumference of each circle.
+
+            * lon_n - Number of circles in the longitudinal direction.
+
+            * lat_n - Number of circles in the latitudinal direction.
+
+        """
+        geod = cartopy.geodesic.Geodesic()
+        geoms = []
+
+        for lat in np.linspace(-80, 80, lat_n):
```

Comment:
> I haven't looked at how well it would work, but I'd also consider putting the tissot on each of the projections in the projection list. (see https://github.com/SciTools/cartopy/blob/master/docs/make_projection.py#L97)

Having had a fiddle, it seems that it would definitely be possible to put indicatrices on each of the projections - it definitely adds to the images, making distortions more clear in a lot of cases.

There is one thing bugging me, which is that the convention for the indicatrices is to place them on gridline intersections like this:
![tissot_world_from_space](https://cloud.githubusercontent.com/assets/13048342/9762154/6533ecac-56f9-11e5-9855-69a24b7c777c.png)

I can't work out if there is  a way to get at these intersections. As far as I can tell, when you call `ax.gridlines()` unless you specify it, the spacing of the gridlines is shipped off to matplotlib land and I'm not really sure how to get at it.. Any thoughts?